controller/AdminAccessController.php

```diff
@@ -220,6 +234,27 @@ protected function validatePermissions($usersPrivileges)
         return false;
     }
 
+    /**
+     * Validates CSRF token and revokes token on success
+     *
+     * @return bool

```

Comment:
Could you please update the phpdoc since the method returns the a token and not a boolean.